tools/spanner/purge_ttl.py

```diff
@@ -5,12 +5,23 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import os
+import sys
+import logging
+from datetime import datetime
+from statsd.defaults.env import statsd
 from urllib import parse
 
 from google.cloud import spanner
 
-# Change these to match your install.
+# set up logger
+logger = logging.getLogger()
```

Comment:
nit: FYI alternatively there's `logging.basicConfig` which can do pretty much all this in one function call